### PR TITLE
Catch error on user not found

### PR DESCRIPTION
**Pull request by Ikinon**
# Pull request

 - [x] I have read and agreed to the [code of conduct](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CONTRIBUTING.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [GitHub projects](https://github.com/orgs/CascadeBot/projects).
 - [x] I have tested all of my changes.
 
## Added/Changed
Changed ban command to catch errors when the user is not found

### CODE REVIEW

File: src/main/java/org/cascadebot/cascadebot/commands/moderation/BanCommand.java [hunk 1]
```diff
@@ -42,14 +43,14 @@ public void onCommand(Member sender, CommandContext context) {
         if (targetMember == null) {
             // If the member is null, the user does not exist in the guild.
             // This attempts to retrieve the user from Discord.
-            targetUser = DiscordUtils.getUser(context.getGuild(), context.getMessage(0), true);
-
-            if (targetUser == null) {
+            try {
+                targetUser = DiscordUtils.getUser(context.getGuild(), context.getMessage(0), true);
+            } catch (ErrorResponseException | NumberFormatException e) {
```
Comment (binaryoverload) [inline src/main/java/org/cascadebot/cascadebot/commands/moderation/BanCommand.java]: We should really be editing DiscordUtils to handle this

Comment (Ikinon) [reply src/main/java/org/cascadebot/cascadebot/commands/moderation/BanCommand.java]: Yea weeryan said that in the last PR that I did, but you said for beta. Are we doing that here now?

File: src/main/java/org/cascadebot/cascadebot/utils/DiscordUtils.java [hunk 2]
```diff
@@ -67,12 +68,15 @@ public static User getUser(Guild guild, String search, boolean retrieve) {
         Checks.notBlank(search, "user");
         List<User> users = FinderUtil.findUsers(search, guild.getJDA());
         if (users.size() > 1) {
-            /*context.replyDanger("There is more than one user that matches this criterion! Please enter the ID or the user's full name!");*/
```
Comment (binaryoverload) [inline src/main/java/org/cascadebot/cascadebot/utils/DiscordUtils.java]: When I said uncomment this, I didn't mean delete it 😂

Comment (Ikinon) [reply src/main/java/org/cascadebot/cascadebot/utils/DiscordUtils.java]: We don't have context passed through so we can't reply like that
*binaryoverload said to comment it back in & say why on discord

**Pull request merged**